app/demo-hifive-inventor/app.toml

```diff
@@ -6,25 +6,41 @@ stacksize = 896
 
 [kernel]
 name = "demo-hifive-inventor"
-requires = { flash = 16752, ram = 3232 }
+requires = { flash = 16784, ram = 3472 }
 features = []
 
 [tasks.jefe]
 name = "task-jefe"
 priority = 0
 max-sizes = { flash = 16384, ram = 2048 }
 start = true
-features = ["semihosting-riscv"]
+features = ["log-stringbuf"]
 stacksize = 1536
 
+[tasks.ext_int_ctrl]
+name = "drv-riscv-plic"
+priority = 1
+max-sizes = { flash = 4096, ram = 2048 }
+start = true
+features = ["log-stringbuf"]
+uses = ["plic"]
+interrupts = { "plic.irq" = 0x1 }
+
+[tasks.ext_int_ctrl.config]
+ints = [52]
+tasks = ['"rtc"']
+notification = [1]
+base = 0x0C00_0000
```

Comment:
Shouldn't this base come from a device "plic" in chip.toml?

Reply:
The `chip.toml` isn't easily available to a task's `build.rs`, but I've already discussed this with @beezow and will be adding it in the next revision.

Reply:
Ack.